README.md

```diff
@@ -2,33 +2,99 @@
 
 Scratch work for porting spectroperfectionism extractions to GPUs
 
-# Directions to run cpu version (currently only working version)
+# Example usage at NERSC
 
-## Set up the environment; there are multiple correct ways to do this
+## CPU
 
 ```
-source /global/cfs/cdirs/desi/software/desi_environment.sh master
-git clone https://github.com/sjbailey/gpu_specter
+# these instructions assume we're at the top level of this repo
 cd gpu_specter
-git checkout refactor
+
+# request an interactive node
+salloc -N 1 -C haswell -t 60 -q interactive
+
+# setup environment
+source /global/cfs/cdirs/desi/software/desi_environment.sh master
 export PATH=$(pwd)/bin:$PATH
 export PYTHONPATH=$(pwd)/py:$PYTHONPATH
-```
 
-## Get an interactive cpu node
+# prepare command line arguments
+basedir=/global/cfs/cdirs/desi/spectro/redux/andes
+args="-w 5760.0,7620.0,0.8 -i $basedir/preproc/20200219/00051060/preproc-r0-00051060.fits -p $basedir/exposures/20200219/00051060/psf-r0-00051060.fits"
 
+# run spex
+srun -n 32 -c 2 bin/spex --mpi -o $SCRATCH/spex_haswell_mpi32_gpu0.fits $args
+
+# for comparison, run desi_extract_spectra
+srun -n 32 -c 2 desi_extract_spectra --mpi -o $SCRATCH/desi_haswell_mpi32_gpu0.fits $args
 ```
-salloc -N 1 -C haswell -t 60 -q interactive
-basedir=/global/cfs/cdirs/desi/spectro/redux/daily/
+
+## GPU
+
+Use the [these instructions](doc/how-to-build-gpu-conda-env.md) for building a gpu conda environment.
+
+### Single GPU
+
 ```
+cd gpu_specter
+module load python esslurm cuda
```

Comment:
For our future selves, I would show exactly which version of cuda we're loading i.e. `module load cuda/10.2.89`. 

We'll need to continue to load 10.2 even after the default cuda module changes.

Reply:
Great point. I added the specific cuda version